Core/src/IfcGeometryConverter/GeomUtils.cpp

```diff
@@ -127,27 +127,15 @@ void GeomUtils::extrude(
 {
 	if( face_loops_input.size() == 0 )
 	{
-		std::cout << "extrude: face_loops_input.size() == 0" << std::endl;
-		return;
-		/*
-		throw oip::InconsistentModellingException("face_loops_input.size() == 0");
-		*/
+		throw oip::InconsistentModellingException("face_loops_input.size() == 0");	

```

Comment:
Please include the name of the function in the messages (similarly to how it was):
```suggestion
		throw oip::InconsistentModellingException("GeomUtils::extrude(): face_loops_input.size() == 0");	
```

Reply:
Also, similarly to other messages, write out, that `face_loops_input.size() should not be 0`.

Reply:
Resolved:  8eb52be